contrib/dualstack.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+if [ ! -z "$1" ] ; then
+  KIND_CONFIG_LCL=$1
+else
+  exit 1
+fi
+
+# Update ${KIND_CONFIG_LCL} for Dual Stack. Move to seperate script.
+# Add "  ipFamily: DualStack" to the networking section of kind.yaml file,
+sed -i '/apiServerPort:.*/a \ \ ipFamily: DualStack' ${KIND_CONFIG_LCL}
+
+# Copy dual stack settings into tmp file. 'sed' can only paste
+# in multiple lines from a file.
+cat > dualstack.txt <<END
```

Comment:
instead of using sed, you could just use python to load the yaml into a dict, modify it, and write it back out. We already use python for the j2 rendering stuff so I don't think its a big deal to use it here as well.

Reply:
I know little python, but no power user. It might take me a little time to figure it out. Is that something that could be a follow-up fix?

Reply:
So I originally spent half a day trying to write a python script that reads the files and parses and edits in python. Once I figured out the j2, made it so much cleaner.

Only issue is that j2 is not loaded in KIND (it is in the daemonset.sh which is inside the container). I added it the the kind.sh. Let me know if that is a big issue or not.

Reply:
its fine. like the j2 render now.